src/Microsoft.AspNetCore.Http.Connections.Client/HttpConnectionOptions.cs

```diff
@@ -8,23 +8,54 @@
 using System.Net.WebSockets;
 using System.Security.Cryptography.X509Certificates;
 using System.Threading.Tasks;
+using Microsoft.AspNetCore.Http.Connections.Internal;
 
 namespace Microsoft.AspNetCore.Http.Connections.Client
 {
-    public class HttpOptions
+    public class HttpConnectionOptions
```

Comment:
There was talk about renaming this. Having a `HttpConnectionOptions` to configure `HttpConnection` makes sense to me.

A `HttpConnectionClientOptions` to configure `HttpConnection` doesn't make as much sense. I think if we rename one we need to rename the other.

Reply:
Yeah, it does seem weird to rename this type. We'll have to thing about it a bit. There's no user-facing "HttpConnection" type on the server, so it makes sense to rename that one to something else.